src/node.ts

```diff
@@ -0,0 +1,29 @@
+import { OsmChange, OsmNode } from '@map-colonies/node-osm-elements';
+import { DEFAULT_ID } from './constants';
+import { createEmptyChange } from './helpers';
+import { Actions, CreateNodeArgs, OsmPoint } from './models';
+
+export const createNode = (args: CreateNodeArgs): OsmNode => {
+  const { lat, lon, id, tags, version } = args;
+  return { id: id ?? DEFAULT_ID, lon, lat, version: version ?? 0, type: 'node', tags };
```

Comment:
You have constants file so why to use `'node'`, put it there.

Reply:
I am not sure how much putting this value in constants will actually achieve.
type is already a closed set of values determined and enforced by typescript.